client/index.jsx

```diff
@@ -117,7 +121,10 @@ Promise.all([rootElemPromise])
 
     store.dispatch(action)
     try {
-      await sessionPromise
+      const [config] = await Promise.all([configPromise, sessionPromise])
+      const serverOrigin = getServerOrigin().toLowerCase()
+      const publicAssetsUrl = new JsonLocalStorageValue(`${serverOrigin}:publicAssetsUrl`)
```

Comment:
Small thing, but I think I would store the entire config here probably, rather than just this one piece of it. Can't really imagine a config value we wouldn't want to cache